src/v/storage/disk_log_impl.cc

```diff
@@ -9,6 +9,7 @@
 
 #include "storage/disk_log_impl.h"
 
+#include "cluster/namespace.h"
```

Comment:
I think we do not want `storage` to depend on `cluster` module.

Reply:
correct!